docs/configuration.md

```diff
@@ -138,3 +138,19 @@ The AWS Account ID and Region is primarily used to construct the ECR domain `[AC
         accountId: 123456789
         region: ap-southeast-2
     ```
+
+### RepositoryTags
```

Comment:
I would prefer to have the documentation mirror the configuration options for easier navigation. Thoughts?

```suggestion
### ECR Options

#### Tags
```